src/containers/ManifestForm.js

```diff
@@ -17,6 +17,12 @@ const mapDispatchToProps = { fetchManifest: actions.fetchManifest };
  * @param theme
  */
 const styles = theme => ({
+  buttons: {
+    textAlign: 'right',
+    '@media (min-width: 600px)': {
```

Comment:
In MUI contexts, I think we can use their theme breakpoint stuff (600px is `small`). There should be other examples of this... uh.. somewhere.

Reply:
https://github.com/ProjectMirador/mirador/blob/master/src/containers/WorkspaceControlPanel.js#L15

Reply:
👍 updated to use MUI's breakpoints in the theme.